src/RottenOranges.java

```diff
@@ -0,0 +1,60 @@
+// LeetCode 994
+import com.google.common.collect.ImmutableList;
+import java.util.LinkedList;
+import java.util.Queue;
+
+class RottenOranges {
+    private int[][] grid;
+    private int numRows;
+    private int numCols;
+    private int numOranges = 0;
+    private int numRottenOranges = 0;
+    private final Queue<Coordinate> rottenOranges = new LinkedList<>();
```

Comment:
```suggestion
    private Queue<Coordinate> rottenOranges = new LinkedList<>();
```
this shouldn't be final. right?